charmcraft/const.py

```diff
@@ -73,7 +74,6 @@
         BaseName("ubuntu", "18.04"),
         BaseName("ubuntu", "20.04"),
         BaseName("ubuntu", "22.04"),
-        BaseName("ubuntu", "23.10"),
         BaseName("ubuntu", "24.04"),
```

Comment:
Are you intentionally not adding 24.10 as a supported base here?

Reply:
Yeah - this should use devel build-base for now.